docs/self-hosted-getting-started.md

```diff
@@ -0,0 +1,99 @@
+# Quick Start: Run a self-hosted database instance with automatic vectorization
+
+If you want to see how it works, follow these steps to setup 
+a self-hosted database instance with automatic embedding vectorization:
+
+## Pre-requisites
+- Docker installed on your machine
+- An OpenAI API key. You can create one [here](https://platform.openai.com/api-keys).
+
+## Setup
+
+1. Run the following docker-compose file to start a TimescaleDB instance and the Vectorizer worker: 
+    ```yaml
+    services:
+      db:
+        image: timescale/timescaledb-ha:cicd-024349a-arm64
+        environment:
+          POSTGRES_PASSWORD: postgres
+          OPENAI_API_KEY: your-api-key
+        ports:
+          - "5432:5432"
+        volumes:
+          - ./data:/var/lib/postgresql/data
+    
+      vectorizer:
+        image: timescale/pgai-vectorizer-worker:rc
+        environment:
+          VECTORIZER_DB_URL: postgres://postgres:postgres@db:5432/postgres
+          OPENAI_API_KEY: your-api-key
+        depends_on:
+          - db
+    ```
+
+2. Connect to your db instance with your DB client of choice. We need to enable the pgai extension and create a simple blog table with the following schema:
```

Comment:
We should add a command to connect directly, for those that don't have a preference or any program installed. If we set `name: pgai`, then we can connect to the db with:

```
docker exec -it pgai-db-1 psql -U postgres
```

Reply:
Done 👍